packages/core/src/components/navigation/Navbar/Navbar.tsx

```diff
@@ -110,11 +110,11 @@ function Navbar({
                 aria-label={menuIconAlt}
               />
               <Link
-                href="/"
                 data-fs-navbar-logo
+                href={logo.link ? logo.link : '/'}
                 prefetch={false}
-                title={homeLabel}
-                aria-label={homeLabel}
+                title={logo.link ? null : homeLabel}
```

Comment:
Why are you setting the `title` to null if there's a logo link? Is that some accessibility recommendation? If that's the case why are you **not**, doing it on `NavbarSlider`?

Reply:
We defaulted "Go to Home" as title if directing to "/". But if the user sets the link via CMS, this text won't be right, since we don't have this info, I removed this prop. But I'm wondering if we should have a new field for this info 🤔

Reply:
I'd go with the new field route 😢

Reply:
After discussing a little bit with @renatamottam, I'll add a new field for it!

Reply:
Added! 🚀